### ISSUE TITLE
List View's size-to-fit doesn't work always

### ISSUE THREAD

**Issue by MarekKnapek** [winsiderss/systeminformer#1689]
### Brief description of your issue

If I double click on list view's header divider, sometimes the column doesn't size to fit.

### Steps to reproduce (optional)

 - Start System Informer.
 - Go to network tab.
 - Select some network connection, right click and select traceroute from context menu.
 - New window appears, most part of the window is dedicated to list view like control.
 - The control has many columns, the last one is country column.
 - As I don't have country database installed, all rows of that column shows text that I'm missing that database.
 - But not really, the column is too narrow, so I cannot read it. It displays beginning of that text followed by an ellipsis (three dots).
 - OK, over the years using Windows operating system I learned that I can double click the header divider to size-to-fit the affected column in order to read it..
 - But that does not work.
 - I'm frustrated as the app is trying to tell me something and I don't see/know what it is. Standard way of adjusting does'n work. I need to over-resize that column in order to find out that I over-resized it and then shrink it back to some acceptable size. It is even more work in remote session (RDP) as the resizing happens only after I let go of mouse button, it doesn't happen during mouse dragging.
 - Digging thru the source code I found out that there is bailout of the shrink-to-fit operation at [treenew.c#L3232](https://github.com/winsiderss/systeminformer/blob/master/phlib/treenew.c#L3232).
 - Maybe you could custom-draw all rows in that column into some dummy DC, then measure max width and then adjust size of that column? I'm not sure.

### Expected behavior (optional)

Double-click on header divider of list-view-like window control should size-to-fit affected column.

### Actual behavior (optional)

No size-to-fit action happens, nothing happens.

### Environment (optional)

```shell
System Informer latest nightly.
```

**Comment by dmex** [comment]
> custom-draw all rows in that column into some dummy DC, then measure max width and then adjust size of that column? I'm not sure.

There's no guarantee the column has text since some columns are graphs and others are images.

We could add another notification message requesting a size but this would only work if there's text?
https://github.com/winsiderss/systeminformer/blob/4e4d1e09264b5082974a8f7e2a6c0db61c72aa45/plugins/NetworkTools/tracetree.c#L449-L494

**Comment by MarekKnapek** [comment]
> There's no guarantee the column has text

Yes, I'm aware of this. I guess the column should contain flags of countries as pictures.

My idea is: If you are able to draw picture, then sure you know dimensions of such picture, don't you? Then, if you know size of such picture, it is easy to compute the biggest one, then it is possible to resize the column.

Anyway, I appreciate how much effort went to System Informer, it is very great tool. Now, I also appreciate the custom tree with columns control, very nice. Thank you. The lack of auto size is just minor inconvenience.
